_posts/2019-02-16-supervirus.html

```diff
@@ -0,0 +1,56 @@
+---
+layout: supervirus
+title: "Supervirus"
+date: 2019-02-16 12:00:00 -0500
+categories: games
+---
+
+<base href={{ "/assets/supervirus/" | relative_url }}>
```

Comment:
vCool. Will certainly need for the future ones

Reply:
Yeah this is str8 🔥 @samgqroberts